### ISSUE TITLE
Accessibility: servicesOverview autocomplete not accessible

### ISSUE THREAD

**Issue by ndarilek** [loconomics/loconomics#635]
Similar to other non-Google autocompletes on the site, and will probably be resolved once you've standardized autocomplete implementations. When I enter a service attribute/tag, I receive no feedback that anything is happening regarding choosing between suggestions.

**Comment by IagoSRL** [comment]
I implemented this, it's at the is528 branch, let me know @ndarilek if you found some accessibility problem or another bug.

**Comment by IagoSRL** [comment]
Hi @ndarilek, additionally to the service attributes autocomplete, I did changes to ensure the buttons that allow to 'remove' an attribute from the list of previously selected ones is accessible. Is not exactly the autocomplete, but is very related to that.

**Comment by ndarilek** [comment]
Confirmed that the autocomplete is now accessible and that the Remove button does indeed work. Nice! Probably good to close this since it seems to resolve my original issue.

**Comment by IagoSRL** [comment]
Good, thanks!
